bitrise.yml

```diff
@@ -26,17 +26,6 @@ workflows:
     steps:
     - activate-ssh-key@4:
         run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
-    - git::https://github.com/DamienBitrise/bitrise-test-plan-sharder.git@master:
```

Comment:
We are not using this step at all. I could not get it working, so I modified the test plans manually.